pkg/gadgets/trace/capabilities/types/types.go

```diff
@@ -38,7 +38,7 @@ type Event struct {
 	Comm      string `json:"comm,omitempty" column:"comm,maxWidth:16"`
 	UID       uint32 `json:"uid,omitempty" column:"uid,minWidth:6"`
 	Cap       int    `json:"cap,omitempty" column:"cap,width:3,fixed"`
-	CapName   string `json:"capName,omitempty" column:"capName,width:20,ellipsis:start"`
+	CapName   string `json:"capName,omitempty" column:"capName,minWidth:18,maxWidth:18,ellipsis:start"`
```

Comment:
```suggestion
	CapName   string `json:"capName,omitempty" column:"capName,width:18,fixed"`
```

`minWidth:18,maxWidth:18` means to have a fixed width. With this change, `ellipsis` is not needed anymore as it will never be truncated. @flyth

Reply:
It could still be truncated if there's no more room anywhere else - however, removing the ellipsis parameter will let it fall back to the default (which should be `end`), which makes more sense now that the common prefix of `CAP_` has been removed.

Reply:
Ok, I marked it as fixed, and I removed the ellipsis to use the default (`end`).